stock_package_autoload/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Stock Package Autoload",
+    "summary": """
+        Select a package to add its content to a transfer line
+    """,
+    "version": "16.0.1.0.0",
+    "author": "PyTech SRL, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": [
+        "stock",
+        "web_domain_field",
```

Comment:
**issue**: this module is deprecated, the same can be achieved in base Odoo by using a Binary field. See: https://github.com/OCA/web/blob/16.0/web_domain_field/README.rst

Reply:
Sorry on missing to notice the deprecation. Thank you for noticing it.